app/hypercerts/[hypercertId]/page.tsx

```diff
@@ -47,17 +52,32 @@ export default async function HypercertPage({ params, searchParams }: Props) {
   const { hypercertId } = params;
 
   const [hypercert] = await Promise.all([getHypercert(hypercertId)]);
+  const isCreatorFeedEnabledOnChain = await creatorFeedFlag();
+  const isEvaluationsEnabledOnChain = await evaluationsFlag();
+  const isMarketplaceListingsEnabledOnChain = await marketplaceListingsFlag();
 
   if (!hypercert) {
     return (
       <ErrorState message="Hypercert not found" hypercertId={hypercertId} />
     );
   }
 
+  const defaultAccordionItems = isMarketplaceListingsEnabledOnChain
+    ? ["item-3"]
```

Comment:
Might be a good moment to rename this to something more meaningful then `item-3`. Which would become confusing as soon as something new gets added. Sneak in the improvements, so to speak ;)

Reply:
Also, is there perhaps a better way than this 3-way ternary operator? This isn't very readable.

Reply:
Done in latest push

Reply:
I'm lost in GitHub's UI. But pushed the refactor to the ternary operator as well